frontend/src/libs/components/input/input.tsx

```diff
@@ -1,10 +1,14 @@
+import 'react-phone-input-2/lib/style.css';
+
 import {
   type Control,
   type FieldErrors,
   type FieldPath,
   type FieldValues,
   type UseFormSetError,
+  Controller,
 } from 'react-hook-form';
+import PhoneInput from 'react-phone-input-2';
 
 import { type InputType } from '~/libs/enums/enums.js';
 import { IconName } from '~/libs/enums/icon-name.enum.js';
```

Comment:
import { type InputType, IconName } from '~/libs/enums/enums.js';
here and in other 10 places all over the project.